src/routes/members/[studentId]/UpdateMemberForm.svelte

```diff
@@ -46,12 +46,17 @@
       error={$errors.lastName}
     />
   </div>
-  <div class="flex gap-2 [&>*:nth-child(3)]:flex-1">
-    <Labeled label="Program" id="classProgramme" error={$errors.classProgramme}>
+  <div class="flex w-full flex-wrap gap-2 [&>*:nth-child(3)]:flex-1">
+    <Labeled
+      label="Program"
+      id="classProgramme"
+      error={$errors.classProgramme}
+      fullWidth={true}
+    >
       <select
         id="classProgramme"
         name="classProgramme"
-        class="select select-bordered w-full max-w-xs"
+        class="w-xs select select-bordered w-full"
```

Comment:
`w-xs` can be replaced by a Tailwind class:
```suggestion
        class="select select-bordered w-full md:max-w-20"
```

Reply:
Yes, that's what I wanted to do but was met with this error: ```Attributes should not contain ':' characters to prevent ambiguity with Svelte directives(illegal-attribute-character) ```